scripts/check-links.js

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
```

Comment:
I think I added these script files to .prettierignore (before @aacebo made the change to eslint) - could you validate whether the scripts in this folder still need to be added or can be removed? At the least I think the .min.js file should remain ignored.

Reply:
Looks like these scripts aren't ignored. That's probably ok tbh